src/components/common/ClientIntakeForm/IntakeForms/DocumentUpload.js

```diff
@@ -0,0 +1,244 @@
+import React, { useState, useEffect } from 'react';
+import { Select, Upload, Button, Table, Space } from 'antd';
+import { connect } from 'react-redux';
+import { document } from '../../../../state/actions';
+import {
+  InboxOutlined,
+  DeleteOutlined,
+  CheckCircleOutlined,
+  ExclamationCircleOutlined,
+} from '@ant-design/icons';
+import axios from 'axios';
+
+const documentOptions = [
+  {
+    label:
+      'Completed the HFCA (Homeless Families Coordinated Assessment) within the past 90 days',
+    value: 'completed_hfca',
+  },
+  {
+    label: 'A valid Driver’s License or State Identification Card',
+    value: 'valid_driver',
+  },
+  {
+    label: 'A valid social security card for everyone in my household',
+    value: 'valid_social',
+  },
+  {
+    label: 'Signed up for benefits through DSHS (WIC, TANF, SNAP)',
+    value: 'dshs_wic_tanf_snap',
+  },
+  {
+    label: 'Completed the Responsible Renters Course',
+    value: 'responsible_renters_course',
+  },
+  {
+    label: 'Birth certificates for children in household',
+    value: 'birth_cert_for_children',
+  },
+  { label: 'Children Enrolled in School', value: 'child_enrolled_school' },
+  { label: 'Childcare', value: 'childcare' },
+];
+
+const documentTypeFilter = [
+  { text: 'completed_hfca', value: 'completed_hfca' },
+  { text: 'valid_driver', value: 'valid_driver' },
+  { text: 'valid_social', value: 'valid_social' },
+  { text: 'dshs_wic_tanf_snap', value: 'dshs_wic_tanf_snap' },
+  { text: 'responsible_renters_course', value: 'responsible_renters_course' },
+  { text: 'birth_cert_for_children', value: 'birth_cert_for_children' },
+  { text: 'child_enrolled_school', value: 'child_enrolled_school' },
+  { text: 'childcare', value: 'childcare' },
+];
+
+const Documents = props => {
+  const [disabled, setDisabled] = useState(true);
+  const [documentType, setDocumentType] = useState(null);
+
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+    },
+    {
+      title: 'Type',
+      dataIndex: 'documentType',
+      key: 'type',
+      filters: documentTypeFilter,
+    },
+    {
+      title: 'Options',
+      key: 'options',
+      render: (text, record, i) => (
```

Comment:
Great idea to add an icon button for the delete function

Reply:
Thanks!